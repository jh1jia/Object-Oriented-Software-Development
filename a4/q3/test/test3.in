stdin count 1 dropfirst 1 dropfirst 2 count l doublewords doublewords allcaps
Hello World
