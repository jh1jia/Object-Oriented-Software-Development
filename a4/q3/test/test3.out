1O
1O
1O
1O
2D
2D
2D
2D
